Sources/Public/BacktraceCrashReporter.swift

```diff
@@ -52,19 +56,49 @@ extension BacktraceCrashReporter: CrashReporting {
         try reporter.enableAndReturnError()
     }
     
+    // This function retrieves, constructs, and sends the pending crash report
     func pendingCrashReport() throws -> BacktraceReport {
         let reportData = try reporter.loadPendingCrashReportDataAndReturnError()
         let attributes = (try? AttributesStorage.retrieve(fileName: BacktraceCrashReporter.crashName)) ?? [:]
-        // NOTE: - no attachments in crash reports
-        return try BacktraceReport(report: reportData, attributes: attributes, attachmentPaths: [])
+        let attachmentPaths = copiedFileAttachments.map {$0.path}
+        return try BacktraceReport(report: reportData, attributes: attributes, attachmentPaths: attachmentPaths)
```

Comment:
I don't think so. The crash report will contain a plist with its own attachment paths from the previous session. If we add attachments user declared in the current user session, we will potentially send double attachments, one from previous session and one from current user session. It could be a cool feature but maybe it can cause confusion